openvdb/doc/changes.txt

```diff
@@ -8,31 +8,40 @@
 
 @par
 Improvements:
-- FindOpenVDB.cmake now correctly propagates CXX version requirements.
+- <TT>FindOpenVDB.cmake</TT> now correctly propagates @c CXX version
+  requirements.
 - Improved directory and file path lookups of some CMake commands in
-  the root CMakeLists.txt <I>[Reported by Daniel Elliott]</I>
+  the root <TT>CMakeLists.txt</TT>.
+  <I>[Reported&nbsp;by&nbsp;Daniel&nbsp;Elliott]</I>
 - @vdblink::tree::LeafNode::modifyValue() LeafNode::modifyValue@endlink and
   @vdblink::tree::LeafNode::modifyValueAndActiveState()
-  LeafNode::modifyValueAndActiveState@endlink now in-place modify voxel values
-  for improved performance.
+  LeafNode::modifyValueAndActiveState@endlink now modify voxel values
+  in place for improved performance.
 
 @par
 Bug fixes:
-- Fixed a bug in @vdblink::tools::computeScalarPotential() computeScalarPotential
-  @endlink that could produce a corrupt result due to invalid memory access.
-  <I>[Reported by Edwin Braun]</I>
+- Fixed a bug in
+  @vdblink::tools::computeScalarPotential() computeScalarPotential@endlink
+  that could produce a corrupt result due to invalid memory access.
+  <I>[Reported&nbsp;by&nbsp;Edwin&nbsp;Braun]</I>
 
 @par
 Python:
-- Updated python module CMake to always produce a .so for the python plugin
+- CMake now always produces a <TT>.so</TT> for the Python module
   on UNIX platforms.
-- Python 3 support improvements to python/pyOpenVDBModule.cc
-  <I>[Reported by yurivict]</I>
+- Fixed a compile-time error when building the Python module for Python&nbsp;3.
+  <I>[Reported&nbsp;by&nbsp;yurivict]</I>
```

Comment:
@pcucka thanks for improving these. bar the re-wording, what should I be doing on my end to catch these issues? Referring to the new lines you've added and the use of &nbsp

Reply:
They're just stylistic preferences.  For text that is not primarily intended to be human-readable, I like to start each sentence or record on a new line.  That makes diffs easier to read, since it limits the scope of each change.

The non-breaking spaces are just to avoid ugliness like "Python" and "3" being split across lines.